src/react/nossr.ts

```diff
@@ -19,8 +20,8 @@ export function useStore<State>(store: Store<State>): State {
   return useStoreBase(store)
 }
 
-export function useUnit(shape) {
-  return useUnitBase(shape)
+export function useUnit(shape, opts?: {forceScope?: boolean}) {
+  return useUnitBase(shape, opts?.forceScope ? getScope() : undefined)
```

Comment:
I suggest to add isomorphic behaviour here something like this:
```suggestion
  return useUnitBase(shape, getScope(opts?.forceScope))
```